### PR TITLE
Add data manipulation and visualization notebook

### PR DESCRIPTION
**Pull request by Shreyanand**
In this PR, I add a notebook for data manipulation and visualization addressing #5.

### CODE REVIEW

**Review by 4n4nd — CHANGES_REQUESTED** [review]
1 minor spelling error, otherwise LGTM :+1:

**Comment by Shreyanand** [comment]
> 1 minor spelling error, otherwise LGTM

Fixed.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+prometheus-api-client
```
Comment (tumido) [inline requirements.txt:1]: @Shreyanand not sure, if I missed the reason to include `requirements.txt` in here but I'd suggest you to include only either `Pipfile.*` or `requrements.txt` (I would favor Pipfile), the other one is redundant.

**Review by MichaelClifford — APPROVED** [review]
LGTM

File: requirements.txt (hunk 1 above)
Comment (Shreyanand) [reply requirements.txt]: Removed requrements.txt

**Review by tumido — APPROVED** [review]

**Pull request merged**